cocos/core/root.ts

```diff
@@ -342,7 +342,7 @@ export class Root {
      * @param width The new width of the window.
      * @param height The new height of the window.
      */
-    public resize (width: number, height: number) {
+    public resize (windowId: number, width: number, height: number) {
```

Comment:
it's a breaking change for `root.resize` interface
please define the interface as `public resize (width: number, height: number, windowId?: number)`

Reply:
ok